FILES/N_0x10144267dd22866c.md

```diff
@@ -1,29 +1,22 @@
 ---
 ns: FILES
+aliases: ["0x10144267DD22866C"]
 ---
-## _0x10144267DD22866C
+## GET_TATTOO_SHOP_DLC_ITEM_INDEX
 
 ```c
 // 0x10144267DD22866C
-int _0x10144267DD22866C(Hash overlayHash, Any p1, int character);
-```
-
-Returns some sort of index/offset for overlays/decorations.
-
-```
-Character types:
-0 = Michael,
-1 = Franklin,
-2 = Trevor,
-3 = MPMale,
-4 = MPFemale
+int GET_TATTOO_SHOP_DLC_ITEM_INDEX(cs_type(Hash) int character, int collection, int preset);
 ```
 
 ```
 NativeDB Introduced: v2189
 ```
 
 ## Parameters
-* **overlayHash**:
-* **p1**:
-* **character**:
+* **character**: AKA eFaction in shop_tattoo.meta. Which character this tattoo belongs to. See [GET_NUM_TATTOO_SHOP_DLC_ITEMS](#_0x278F76C3B0A8F109).
+* **collection**: Which collection this tattoo belongs to. If you specify -1, it will check all collections.
+* **preset**: Which preset this tattoo belongs to.
+
+## Return value
+Returns the actual index (Not ID from shop_tattoo meta) or -1 (if it failed to find the tattoo) from 0 to [GET_NUM_TATTOO_SHOP_DLC_ITEMS](#_0x278F76C3B0A8F109) of a tattoo.
```

Comment:
Returns the actual index (Not ID from shop_tattoo meta) from 0 to [GET_NUM_TATTOO_SHOP_DLC_ITEMS](#_0x278F76C3B0A8F109) or -1 (if it failed to find the tattoo) of a tattoo.


Structure would be better that way in my opinion.

Reply:
sure